vgstation13.dme

```diff
@@ -1001,6 +1000,7 @@
 #include "code\game\objects\items\devices\pcmc.dm"
 #include "code\game\objects\items\devices\pipe_painter.dm"
 #include "code\game\objects\items\devices\powersink.dm"
+#include "code\game\objects\items\devices\radstorm_remote.dm"
```

Comment:
huh? why move this line?

Reply:
Dream Maker did this automatically. I'm guessing you just stuck this line in the .dme somewhere because dream maker has its own sorting.